.evergreen/ocsp/certs.yml

```diff
@@ -61,8 +61,49 @@ certs:
     subjectAltName:
       DNS: localhost
       IP: 127.0.0.1
-    authorityInfoAccess: 'OCSP;URI:http://localhost:8100/status/'
+    authorityInfoAccess: 'OCSP;URI:http://localhost:9001/power/level/,OCSP;URI:http://localhost:8100/status/'
     mustStaple: true
     subjectKeyIdentifier: hash
     keyUsage: [digitalSignature, keyEncipherment]
     extendedKeyUsage: [serverAuth, clientAuth]
+
+- name: 'ocsp_responder.crt'
+  description: Certificate and key for the OCSP responder
```

Comment:
We previously added the ocsp_responder certs but neglected to update the yml file